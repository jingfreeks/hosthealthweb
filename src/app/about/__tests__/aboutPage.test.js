import { render, screen } from "@testing-library/react";
import AboutPage from "../page";

describe("About Page", () => {
  it("renders About Page to match snaptshot", () => {
    const { container } = render(<AboutPage />);
    expect(container).toMatchSnapshot();
  });
});
