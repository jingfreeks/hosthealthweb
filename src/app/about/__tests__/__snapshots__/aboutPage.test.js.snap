// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page renders About Page to match snaptshot 1`] = `
<div>
  <div>
    About
  </div>
  <a
    href="/"
  >
    Link to Home Page
  </a>
</div>
`;
